python/cudf/cudf/_libxx/cpp/sorting.pxd

```diff
@@ -3,20 +3,40 @@
 from libcpp cimport bool
 from libcpp.memory cimport unique_ptr
 from libcpp.vector cimport vector
+from libc.stdint cimport int32_t
 
 from cudf._libxx.types import np_to_cudf_types, cudf_to_np_types
 
 from cudf._libxx.cpp.column.column cimport column
+from cudf._libxx.cpp.column.column_view cimport column_view
+from cudf._libxx.cpp.table.table cimport table
 from cudf._libxx.cpp.table.table_view cimport table_view
 cimport cudf._libxx.cpp.types as libcudf_types
 
+cdef extern from "cudf/sorting.hpp" namespace "cudf" nogil:
+    ctypedef enum rank_method:
+        FIRST "cudf::rank_method::FIRST"
+        AVERAGE "cudf::rank_method::AVERAGE"
+        MIN "cudf::rank_method::MIN"
+        MAX "cudf::rank_method::MAX"
+        DENSE "cudf::rank_method::DENSE"
+
+ctypedef int32_t underlying_type_t_rank_method
```

Comment:
This shouldn't be in the `_libxx/cpp/sorting.pxd` file. It should go into `_libxx/sort.pxd` since it's not externally defined.

Reply:
moved to `_libxx/sort.pyx`.